### ISSUE TITLE
Add environment variables to notebook instance

### ISSUE THREAD

**Issue by yang-guo** [rationalmatter/Juno-Issue-Tracker#128]
**Is your feature request related to a problem? Please describe.**
Currently, the only way to add environment variables to an active notebook is to use os.environ within the notebook.  While this works, it doesn't allow you to run existing notebooks (e.g. from a repository) that depend on environment variables without adding additional cells and exposing potential secrets.

**Describe the solution you'd like**
Have an option within the menu interface to add/save environment variables that get loaded when any notebook starts up.  This will allow secrets to be local to the app without needing to expose them in any notebooks.

**Comment by navoshta** [comment]
Thanks for reporting @yang-guo, I will look into this.

**Comment by Manouchehri** [comment]
Was this actually ever added?

**Comment by alexstaravoitau** [comment]
@Manouchehri No, not yet—I'll reopen the issue as it shouldn't have been closed really.
